### PR TITLE
PIS etasu met requirements UI

### PR DESCRIPTION
**Pull request by Meng-Chau**
Pull request made in case needed by the end of Monday/start of Tuesday

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -143,5 +143,8 @@
       }
     }
   },
-  "defaultProject": "pro"
+  "defaultProject": "pro",
+  "cli": {
+    "analytics": "b9209b06-52d5-45ae-9131-080bbf4eb3a3"
```
Comment (smalho01) [inline angular.json]: this change idt needs to be here, was probably added by the cli automatically

File: src/app/mainwindow/a-doctor-order-window/new-doctor-order-window/new-doctor-order-item/etasu-popup/etasu-popup.component.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  isLoading = false;
+
+
+  constructor(@Inject(MAT_DIALOG_DATA) public data : {order: any}, private doctorOrderService: DoctorOrderServices, 
+    private snackBar : MatSnackBar, public dialogRef : MatDialogRef<EtasuPopupComponent>) { }
+
+
+  ngOnInit(): void {
+    this.isLoading = true;
+    this.doctorOrderService.getDocOrders();
```
Comment (smalho01) [inline src/app/mainwindow/a-doctor-order-window/new-doctor-order-window/new-doctor-order-item/etasu-popup/etasu-popup.component.ts]: we don't need to call getDocOrders on init

**Review by smalho01 — CHANGES_REQUESTED** [review]

File: angular.json (hunk 1 above)
Comment (Meng-Chau) [reply angular.json]: do you mean that it should just be "defaultProject": "pro" by itself?

Comment (smalho01) [reply angular.json]: yes

**Review by smalho01 — APPROVED** [review]

**Pull request merged**